### PR TITLE
9.3.0 kubernetes

### PR DESCRIPTION
**Pull request by rkhamis**

### CODE REVIEW

File: modules/runtimes/PrefabGolang.py [hunk 1]
```diff
@@ -118,12 +118,19 @@ def clean_src_path(self):
         srcpath = self.prefab.core.joinpaths(self.GOPATHDIR, 'src')
         self.prefab.core.dir_remove(srcpath)
 
-    def get(self, url):
+    def get(self, url, install=True, update=True):
         """
```
Comment (ahussein) [inline modules/runtimes/PrefabGolang.py]: can you please update the docstrings?

File: modules/virtualization/PrefabKubernetes.py [hunk 2]
```diff
@@ -36,11 +35,27 @@ def multihost_install(self, nodes=[], reset=False):
         """
         pass
 
+    def install_dependencies(self, reset=False):
+        """
+        @param reset ,,if True will resintall even if the code has been run before.
```
Comment (ahussein) [inline modules/virtualization/PrefabKubernetes.py]: resintall ? typo!

Comment (ahussein) [reply modules/virtualization/PrefabKubernetes.py]: Would be amazing if we can add a test case to validate the installation!

**Comment by abdulrahmantkhalifa** [comment]
i did not create this PR , this branch is not done

**Pull request closed without merging**